ddp-workspace/projects/ddp-testboston/src/app/components/crc-dashboard/crc-dashboard.component.ts

```diff
@@ -56,6 +57,7 @@ export class CrcDashboardComponent implements OnInit, OnDestroy {
 
   public setSelectedSubject(): void {
     this.sessionService.setParticipant(this.studySubject.userGuid);
+    this.config.invitationId = this.studySubject.invitationId;
```

Comment:
New mockups assume that activity will show the selected subject's invitation id - https://drive.google.com/drive/u/1/folders/10QOZmiT1BYv2OI7F6Gpm19tEw_jeNT57
for this reason, we should store selected invitation id in the config.

Reply:
I feel like the invite code should belong in the `session` instead of the config...

Reply:
I also had this idea but declined it, so, since we both thought about it, I will implement that :)